tests/server/mocha_wrapper.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-export POSTGRES_URL="postgres://localhost/server_test"
+export POSTGRES_URL="postgres://postgres@localhost/server_test"
```

Comment:
I had to add an explicit username here, otherwise the user is arbitrarily determined based on the linux user you are logged into the current linux machine as, which is something that might change from CI run to CI run. This happens to work as well for me locally (I use Postgres.app, and it always includes a `postgres` user in the database), but **it could break for other people.**

If this is a concern, we need to do some work so that this script loads some database configuration from the environment rather than hardcoding it all here.

Reply:
This seems like a step forward to me.  I am happy to merge as is.